src/views/AboutView.vue

```diff
@@ -271,12 +322,55 @@
       },
       goToReviews(){
         this.showInfo =false;
-        this.showReview = true;
+        if(this.resenas.length == 0){
```

Comment:
aca puede causar problemas porque es un loose equal, mejor cambialo a strict === para que no tenga comportamientos raros 
[referencia](https://www.freecodecamp.org/news/loose-vs-strict-equality-in-javascript/#:~:text=The%20%3D%3D%20operator%20performs%20a,well%20as%20the%20same%20value).)

Reply:
listo!, ya lo cambie